README.md

```diff
@@ -1,38 +1,148 @@
-# osde2e
+# OSDe2e
+Comprehensive testing solution for Service Delivery
 
-This project checks OSD releases by starting an OSD cluster and verifying operation through testing
+## Purpose
+Provide a standard for testing every aspect of the Openshift Dedicated product. Use data derived from tests to inform release and product decisions.
 
-## Running
-These steps run the osde2e test suite. All commands should be run from the root of this repo.
+## Execution 
+These steps run the OSDe2e test suite. All commands should be run from the root of this repo.
 
-A properly setup [Go workspace](https://golang.org/doc/code.html#GOPATH) using Go 1.13+ is required.
+A properly setup Go workspace using **Go 1.13+ is required**.
 
-1. Get token to launch OSD clusters [here](https://cloud.redhat.com/openshift/token).
+Get token to launch OSD clusters here.
 
-2. Install dependencies:
-    ```bash
-    # Install dependencies
-    $ go mod tidy
 
-    # Copy them to a vendor dir
-    $ go mod vendor
-    ```
-1. Set `OCM_TOKEN` environment variable:
-    ```bash
-    $ export OCM_TOKEN=<token from step 1>
-    ```
-4. Run tests:
-    ```bash
-    $ go test -v . -test.timeout 4h
-    ```
+Install dependencies:
+```
+# Install dependencies
+$ go mod tidy
+# Copy them to a vendor dir
+$ go mod vendor
+```
 
-## Configuring
-osde2e is configured using a set of environment variables.
-The options available are found [here](./docs/Options.md).
+Set OCM_TOKEN environment variable:
+```
+$ export OCM_TOKEN=<token from step 1>
+```
 
-Common ones are:
-- [`NO_DESTROY`](./docs/Options.md#no_destroy): don't delete clusters after testing
-- [`CLUSTER_ID`](./docs/Options.md#cluster_id): test an existing cluster specified by ID
+Run tests:
```

Comment:
I'm wondering if we should excise this section and label the Test Examples section as "Running the Tests." Reading this again, the Example section basically covers this in much greater detail.

Reply:
Sounds good, I'll shuffle it around and give it another pass.